gammapy/data/observation_summary.py

```diff
@@ -125,3 +126,33 @@ def show_in_browser(self):
         """Make HTML file and images in tmp dir, open in browser.
         """
         raise NotImplementedError
+
+
+# class ObservationSummary(object):
```

Comment:
I try to implement this class and keep it simple. Now it take an ObservationStatsList and do some standard plots of detection. Maybe we could wrap it in an other class taking some options or a configuration file, to deal with the regions and the background substraction, as you suggested if I remembered correctly. We could also add the ObservationTableSummary utilities.